setup

```diff
@@ -6,11 +6,18 @@ set -e
 #
 # --- Based Off of/Ripped from https://github.com/FasterMelee/FasterMelee-installer/ Thank you Faster Melee Team!
 # ---
+ONLINEMASTER=$(curl -Ls https://github.com/Birdthulu/FPM-Installer/raw/master/setup | md5sum)
+CURRENTMASTER=$(md5sum setup)
+##Cut these down to just 32 hex chars
 
-if [ -f "./setup" ] && [ -z "${IN_NIX_SHELL++}" ]; then
-	echo "You are running a hosted copy of the script. Please *delete the local file 'setup'* and run"
+ONLINEMASTER=$(echo $ONLINEMASTER | cut -c -32)
+CURRENTMASTER=$(echo $CURRENTMASTER | cut -c -32)
+echo $ONLINEMASTER
```

Comment:
Needs a check for if the person already has a setup file on their computer or not. If they don't have the file skip the hash check and just continue the script

Reply:
Ok, I avoid the check now if the setup file is not found in the current directory.